frontend/src/components/eMIB/EditEmail.jsx

```diff
@@ -74,7 +76,51 @@ const styles = {
 };
 
 class EditEmail extends Component {
+  static propTypes = {
+    onChange: PropTypes.func.isRequired
+  };
+
+  state = {
+    emailType: EMAIL_TYPE.reply,
+    emailTo: "",
+    emailCc: "",
+    emailBody: "",
+    reasonsForAction: ""
+  };
+
+  onEmailTypeChange = event => {
+    const newEmailType = event.target.value;
+    this.setState({ emailType: newEmailType });
+    this.props.onChange({ ...this.state, emailType: newEmailType });
+  };
+
+  onemailToChange = event => {
+    const newemailTo = event.target.value;
+    this.setState({ emailTo: newemailTo });
+    this.props.onChange({ ...this.state, emailTo: newemailTo });
+  };
+
+  onemailCcChange = event => {
+    const newemailCc = event.target.value;
+    this.setState({ emailCc: newemailCc });
+    this.props.onChange({ ...this.state, emailCc: newemailCc });
+  };
+
+  onemailBodyChange = event => {
+    const newemailBody = event.target.value;
+    this.setState({ emailBody: newemailBody });
+    this.props.onChange({ ...this.state, emailBody: newemailBody });
+  };
+
+  onreasonsForActionChange = event => {
```

Comment:
Same comment as my previous comment: the name of the function should be _'onReasonsForActionChange'_ to be consistent.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/180/commits/a30fbec8f08be4778d32f2cc176d05761bbb495b and commit #https://github.com/code-for-canada/project-thundercat/pull/180/commits/ff90c3754c23b30b4ff2fbfcca6ef2cb0277391a